ci/libvcx.dockerfile

```diff
@@ -67,7 +116,7 @@ ARG NODE_VER=8.x
 RUN curl -sL https://deb.nodesource.com/setup_${NODE_VER} | bash -
 RUN apt-get install -y nodejs
 
-RUN chown -R indy . 
+RUN chown -R indy .
 
 USER indy
 
```

Comment:
How about removing the LABELs?

Reply:
That wouldn't reduce size much. Avoiding installing rustup here would save 700mb, but lets leave that for later..(since Ubuntu libvcx might be thrown away soon anyway)